pkgs/coverage/lib/src/formatter.dart

```diff
@@ -129,6 +132,21 @@ extension FileHitMapsFormatter on Map<String, HitMap> {
       buf.write('end_of_record\n');
     }
 
+    // Add uncovered files if allowed
+    for (final file in uncoveredFiles) {
+      if (includeUncovered != null && !includeUncovered(file)) continue;
```

Comment:
If `includeUncovered` is null, then no uncovered files should be included. That's to ensure we don't change behavior for existing users.